src/bootstrap.ts

```diff
@@ -75,22 +74,12 @@ async function connect(delayMillis: number, mongoUrl: string, username: string,
 	try {
 		// https://mongoosejs.com/docs/connections.html
 		await mongoose.connect(mongoUrl, {
-			autoReconnect: true,
-			// http://mongodb.github.io/node-mongodb-native/3.1/reference/faq/
-			socketTimeoutMS: 10000,
-			connectTimeoutMS: 30000,
-			keepAlive: true,
-			reconnectTries: 10,
-			reconnectInterval: 3000,
-			bufferCommands: false,
-			bufferMaxEntries: 0,
+			directConnection: true,
 			user: username,
 			pass: password,
-			// https://mongoosejs.com/docs/deprecations.html
-			useNewUrlParser: true,
-			useFindAndModify: false,
-			useCreateIndex: true,
+			socketTimeoutMS: 10000,
```

Comment:
`useNewUrlParser, `useFindAndModify`: deprecated
https://mongoosejs.com/docs/5.x/docs/deprecations.html

`useCreateIndex`: deprecated
https://mongoosejs.com/docs/migrating_to_6.html#no-more-deprecation-warning-options

`keepAlive`: deprecated
https://mongoosejs.com/docs/connections.html#keepAlive

Reply:
`bufferCommands: false` causes ~17 tests to fail so for now I've removed it